app/settlement.js

```diff
@@ -2,8 +2,9 @@ const db = require('./data')
 
 const updateSettlementStatus = async (returnData) => {
   if (returnData.settled) {
-    await db.completedPaymentRequest.update({ settled: returnData.settlementDate }, { where: { invoiceNumber: returnData.invoiceNumber } })
+    await db.completedPaymentRequest.update({ lastSettlement: returnData.settlementDate, settledValue: db.Sequelize.literal(`COALESCE("settledValue", 0) + ${returnData.value}`) }, { where: { invoiceNumber: returnData.invoiceNumber } })
```

Comment:
I've used COALESCE to ensure that there is a value of 0 if `settledValue` is null.  That shouldn't be the case but just as a failsafe to ensure that the return value can't be ignored.